roles/bootstrap-os/tasks/setup-pipelining.yml

```diff
@@ -1,8 +1,7 @@
 ---
-# Remove requiretty to make ssh pipelining work
-
-- name: Remove require tty
+- name: Remove requiretty to make ssh pipelining work
```

Comment:
I think we can actually remove this task. It should not be required anymore by Ansible.